### ISSUE TITLE
Segmentation fault - More specifics about the package requirements please

### ISSUE THREAD

**Issue by yrf1** [jshang123/G-Bert#3]
Hi, can you provide more details on the installation requirements please? For example, which specific combination of the package versions can be used to run G-Bert for: 
- pytorch
- torch_geometric
- torch_scatter
- torch_sparse
- torch_cluster
- GCC version
(So like not a range of versions, but one example of the combination of specific versions that worked).

Because I am facing package dependency issues. If I use `torch_geometric==1.0.3` I get a segmentation fault in the softmax function line 224 of graph_models.py. But if I use `torch_geometric==1.2.1` I get a dimension mismatch in line 222 of graph_models.py. So it seems to me the code is dependent on package version which more detail would help.

Thank you!

**Comment by yrf1** [comment]
Because if I use `torch_geometric==1.2.0` I get a segmentation fault in the softmax function line 224 of graph_models.py. But if I use `torch_geometric==1.2.1` I get a dimension mismatch in line 222 of graph_models.py. So it seems to me the code is dependent on package version which more detail would help.

**Comment by jshang123** [comment]
Hi, @yrf1. Sorry for the late reply. The detailed installation requirements are as follows:

tqdm==4.31.1
numpy==1.16.2
torch_geometric==1.0.3
pandas==0.24.2
dill==0.2.9
torch==1.2.0
scikit_learn==0.21.3
tensorboardX==1.9
torch_scatter==1.3.1
torch_sparse==0.4.0
torch_cluster==1.4.4
gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)

**Comment by yrf1** [comment]
Thank you!
